src/main/java/io/antmedia/streamsource/StreamFetcherManager.java

```diff
@@ -227,7 +227,16 @@ public void playNextItemInList(String streamId, IStreamFetcherListener listener)
 
 		// Get current playlist in database
 		Broadcast playlist = datastore.get(streamId);
-
+		
+		if(playlist != null && !playlist.isPlaylistLoopEnabled() && playlist.getPlayListItemList().size() <= playlist.getCurrentPlayIndex()+1) {
```

Comment:
I think it's better to make this check in  `skipNextPlaylistQueue` because it resets the index to zero.

Reply:
When starting this work I tried to make it in `skipNextPlaylistQueue` but it's hard to stop the playlist queue.